### PR TITLE
Typo fix and add forgotten built-in for spatial shader docs

### PR DESCRIPTION
**Pull request by paddy-exe**
Added the . at the end of the built-ins I forgot and the built-ins CAMERA_DIRECTIOON_WORLD for the fragment built-ins from https://github.com/godotengine/godot-docs/pull/6022

### CODE REVIEW

File: tutorials/shaders/shader_reference/spatial_shader.rst [hunk 1]
```diff
@@ -273,11 +273,13 @@ these properties, and if you don't write to them, Godot will optimize away the c
...
 +----------------------------------------+--------------------------------------------------------------------------------------------------+
-| in vec3 **NODE_POSITION_VIEW**         | Node view space position                                                                         |
+| in vec3 **NODE_POSITION_VIEW**         | Node view space position.                                                                        |
 +----------------------------------------+--------------------------------------------------------------------------------------------------+
-| in vec3 **CAMERA_POSITION_WORLD**      | Camera world space position                                                                      |
+| in vec3 **CAMERA_POSITION_WORLD**      | Camera world space position.                                                                     |
++----------------------------------------+--------------------------------------------------------------------------------------------------+
+| in vec3 **CAMERA_DIRECTION_WORLD**     | Camera world space direction.                                                                    |
```
Comment (clayjohn) [inline tutorials/shaders/shader_reference/spatial_shader.rst:282]: Since you are already editing these, can you rephrase them like I suggested in [the backport](https://github.com/godotengine/godot-docs/pull/6152#discussion_r965116563)? (really sorry to be nitpicky)
i.e. ``Node world space position.`` becomes ``Node position, in world space.``

**Comment by paddy-exe** [comment]
Thanks for the feedback! I updated the PR accordingly

**Review by clayjohn — APPROVED** [review]

**Comment by clayjohn** [comment]
Thanks! Looks great

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thank you! 🎉
